src/Shortcodes/FileShortcodeProvider.php

```diff
@@ -136,17 +136,30 @@ public static function handle_shortcode($arguments, $content, $parser, $shortcod
      */
     protected static function getCachedMarkup($cache, $cacheKey, $arguments): string
     {
+        // Retrieve the cache markup
         $item = $cache->get($cacheKey);
+        if (empty($item) || empty($item['markup'])) {
+            // This is a cache miss
+            return '';
+        }
+
+        // Check the file exists
+        $filename = $item['filename'];
+        $hash = $item['hash'];
```

Comment:
We don't need to turn these into variables, I'd personally prefer just using the array directly since it's clear what those represent without setting them to variables. That said, I won't hold up merging if you choose to leave them as is.
It would clear up the above point if you just reference them as array items instead of using these variables though, since we'll be checking `empty($item['filename'])` below, removing any chance of ambiguity.

Reply:
My preference is to keep them as variable to make the code to make the code more readable below.

Reply:
I don't think it does make the code more readable - but it's really subjective so I'll let you leave it as is.